healthcare/healthcare/report/diagnosis_trends/diagnosis_trends.py

```diff
@@ -0,0 +1,164 @@
+# Copyright (c) 2013, Frappe Technologies Pvt. Ltd. and contributors
+# For license information, please see license.txt
+
+import frappe
+from frappe import _, scrub
+from frappe.utils import add_days, add_to_date, flt, getdate
+from six import iteritems
+
+from erpnext.accounts.utils import get_fiscal_year
+
+
+def execute(filters=None):
+	return DiagnosisTrends(filters).run()
```

Comment:
Can we add tests to check if correct data is being fetched based on Department, Dates, and Range filters?

Reply:
Sure @ruchamahabal